.github/workflows/lint-pipelines.yml

```diff
@@ -0,0 +1,36 @@
+name: Lint Pipelines
```

Comment:
"pipelines" has a bunch of meanings, if it's separated from its context.
"Lint Deployment Pipelines" would be significantly less ambiguous.

P3